pyoffers/models/core.py

```diff
@@ -77,7 +86,14 @@ def init_instance(self, data):
         return self.model(manager=self, **data)
 
     def _call(self, method, **kwargs):
-        return self.api._call(self.model.__name__, method, **kwargs)
+        return self.api._call(self.target, method, **kwargs)
+
+    @property
+    def target(self):
+        if self.is_application_manager:
```

Comment:
I propose to remove ``is_application_manager`` attribute and use inheritance instead. ``ApplicationManager`` could be placed in ``core.py`` and used as a base class for OfferCategoryManager.

Also, it would be easier to redefine target property later if it will just return ``self.model.__name__``. ``ApplicationManager``, for example, will have ``target = 'Application'``

Reply:
done